cmake/std/PullRequestLinuxDriverTest.py

```diff
@@ -124,12 +135,14 @@ def confirmGitVersion():
 
     if major_git_version  <  2 or (major_git_version == 2 and
                                    minor_git_version < 10):
-        raise SystemExit("Git version  should be 2.10 or better - Exiting!")
```

Comment:
moving this space is what caused the tests to fail. I'm sure that extra space is in the test as well ;->

Reply:
huh? how does an extra whitespace in the text string of the exit cause a failure?

Reply:
It's not a smart comparison so the strings must match exactly.  Not elegant is it?